challenge_01/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+    background-color: #E2F1E7;
```

Comment:
It is virtually always better to keep most values as [CSS Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).
Values like colors, fonts, important measurements, shadows and other things.

Reply:
> It is virtually always better to keep most values as [CSS Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties). Values like colors, fonts, important measurements, shadows and other things.

Not familiar with this concept. Should I learn it?